dataprofiler/labelers/character_level_cnn_model.py

```diff
@@ -74,6 +74,133 @@ def create_glove_char(n_dims: int, source_file: str = None) -> None:
             file.write(word + " " + " ".join(str(num) for num in embd) + "\n")
 
 
+@tf.keras.utils.register_keras_serializable(package="CharacterLevelCnnModel")
+class ThreshArgMaxLayer(tf.keras.layers.Layer):
```

Comment:
Refactored threshargmax out of the class function so it could be properly serialized in keras v3